packages/shared-components/.prettierignore

```diff
@@ -0,0 +1 @@
+src/tailwind.css
```

Comment:
@BShaq Just fyi - I updated the two of the prettier ignore files so that `npm run format` does not format automatically generated files. Otherwise, we will always have 500+ changes for example for the tailwind file in `shared-components` alone. And these will go back and forth with each new generate of the file...

Reply:
Oh, thanks for pointing that out :)